bin/LDA.py

```diff
@@ -143,48 +147,44 @@ def handle_program_options():
                                                  "plots based on normalized relative "
                                                  "abundances or distance matrices "
                                                  "(for e.g. unifrac distance matrix).")
-    parser.add_argument("-i", "--otu_table", required=True,
-                        help="Input biom file format OTU table. [REQUIRED]")
     parser.add_argument("-m", "--map_fp", required=True,
                         help="Metadata mapping file. [REQUIRED]")
     parser.add_argument("-g", "--group_by", required=True,
-                        help="A column name in the mapping file containing\
-                              categorical values that will be used to identify \
-                              groups. Each sample ID must have a group entry. \
-                              Default is no categories and all the data will be \
-                              treated as a single group. [REQUIRED]")
+                        help="A column name in the mapping file containing categorical "
+                             "values that will be used to identify groups. Each sample "
+                             "ID must have a group entry. Default is no categories and "
+                             "all the data will be treated as a single group. [REQUIRED]")
     parser.add_argument("-c", "--colors",
-                        help="A column name in the mapping file containing\
-                              hexadecimal (#FF0000) color values that will\
-                              be used to color the groups. Each sample ID must\
-                              have a color entry.")
-    parser.add_argument("-dm", "--dist_matrix_file",
-                        help="Input distance matrix file.")
+                        help="A column name in the mapping file containing hexadecimal "
+                             "(#FF0000) color values that will be used to color the "
+                             "groups. Each sample ID must have a color entry.")
+    parser.add_argument("-i", "--otu_table", help="Input biom file format OTU table.")
```

Comment:
Now that otu_table is no longer the "default" expected input, should we change from -i to something like -ot to match the -dm argument?